{"name": "我是cate的第四条用例!", "status": "failed", "statusDetails": {"message": "Failed", "trace": "@allure.title(\"我是cate的第四条用例!\")\n    def test_04():\n        print(\"我是cate的第四条用例!\")\n>       pytest.fail()\nE       Failed\n\nTestcase_Api\\Shop\\test_cate.py:22: Failed"}, "attachments": [{"name": "log", "source": "c36abb4a-25de-4eaf-a8e8-fb9fca2d8ab2-attachment.txt", "type": "text/plain"}], "start": 1670495154851, "stop": 1670495154851, "uuid": "425a6083-1930-4316-8001-5b3fd41c5143", "historyId": "ec3cd6a6ccc41a92ae90897b1d080f2a", "testCaseId": "a1d8ad9c228ca0fd62f8507d21f798b9", "fullName": "Testcase_Api.Shop.test_cate#test_04", "labels": [{"name": "parentSuite", "value": "Testcase_Api.Shop"}, {"name": "suite", "value": "test_cate"}, {"name": "host", "value": "LAPTOP-1IKMED1I"}, {"name": "thread", "value": "16572-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase_Api.Shop.test_cate"}]}